base/applications/regedit/find.c

```diff
@@ -97,7 +97,7 @@ CompareData(DWORD dwType, LPCVOID pv1, size_t cb1)
         size_t cch1 = cb1 / sizeof(WCHAR), cch2 = wcslen(s_szFindWhat), i;
 
         if (cb1 & 1)
-            DPRINT1("Odd size for string (%lu)!\n", dwType);
```

Comment:
Most importantly it would be of interest to indicate the name of the corresponding registry **value** (and its path in the registry) so that we can identify these.

Reply:
May be ok as a **temporary** debug thing. Feels a bit like abusing regedit as a debugger though.
But please at least without trailing ! then. :-)

Reply:
In which case, this has to be done only by the interested person, in his **own** local branch/repo, in order for him to investigate what is adding said odd-counted unicode string, and fix (with a PR) that underlying problem.

Reply:
Removed.